returnn/datasets/util/vocabulary.py

```diff
@@ -162,12 +163,43 @@ def get_seq_indices(self, seq):
 
   def get_seq_labels(self, seq):
     """
-    :param list[int] seq:
+    :param list[int]|int|numpy.ndarray seq: 0D or 1D
```

Comment:
Why is `int` or 0D needed?

Reply:
The layer output can be 0D (i.e. if you don't have a time dim). We could either directly map this to 1D when calling this function (e.g. in search) or do it here. I think it's more robust to do it here.

Reply:
I'm not sure this is confusing. The function name mentions `seq`.

I'm also not sure this would complicate the logic here and for all derived functions where you always repeat the same code.

When you just do this when you call the function, I think it is easier.

But maybe also for the case that it is not a sequence (the layer output does not have a time dim), you would not call this function at all, but just do a direct lookup in `labels`?

Similarly, when the layer output has other unrelated dims (maybe a second spatial dim), you would also not directly call this function. I'm not really sure what you would do. It's ok to just throw an exception then.